docs/policies/threat_intelligence_policy.md

```diff
@@ -0,0 +1,348 @@
+
+The DfE will collect threat intelligence from external sources in support of the objectives outlined above, considering factors including (but not limited to):
+
+- relevance of the data (for example, industry sector or threat actor trends)
+- reputation of source for quality and timeliness of data
+- frequency of updates
+
+
+## Sources
+
+Threat intelligence should be gathered from a variety of sources:
+
+- industry and government threat intelligence, such as UK’s National Cyber Security Centre (NCSC), Cyber Security Information Sharing Partnership (CiSP) and industry groups such as Warning Advice and Reporting Points (WARP)
+- commercial threat intelligence feeds (for example, Digital Shadows, Recorded Future, Team Cymru, VirusTotal)
+- internal threat intelligence - Historical data from previous incidents and internal logs and alerts shall be used to enhance threat intelligence and detection
+- open-source intelligence - OSINT (for example, MISP-project.org, Spamhaus, Virusshare, Abuse.ch)
+
+Data collected will include:
+
+- known vulnerabilities for example, Common Vulnerabilities and Exposures (CVE)
+- zero-day exploits from bug bounty and vulnerability reporting processes
+- indicators of compromise (IOCs)
+- emerging malware and phishing tactics
+- known exploits and tactics, techniques and procedures (TTPs) of key threat actors
+- identification of exposed DfE or customer data found on the internet
+- identification of fraudulent sites /cyber-squatting
+
+Agreements should also be established for regular intelligence sharing between government departments
+
+
+## Analysis
+
+Threat intelligence collected should be analysed to:
+
+- aggregate and eliminate duplicate data
+- assess the credibility, relevance, reputation and context of the collected intelligence
+- prioritise the intelligence, in a timely manner, based on criticality and potential impact
+
+
+## Communication / distribution
+
+The prioritised and targeted threat intelligence must then be distributed to the relevant stakeholder groups:
+
+- internal teams identified in the responsibility section in this policy
+- external partners and third parties (as permitted by data sharing agreements)
+
+The distributed threat intelligence should be clear, concise and provide appropriate information to:
+
+- raise and maintain awareness of the threats to DfE 
+- provide detail that enables teams to take appropriate action. For example, record and manage security incidents
+
+
+## Threat intelligence use
+
+DfE teams should actively use threat intelligence to help manage risk in the following activities.
+
+
+### Risk management
+
+Threat Intelligence should be used to inform the Threat Assessment and following Risk Assessment activities of the DfE and its projects. Threats identified from the threat assessment shall be fed back to the threat intelligence teams for inclusion into the threat intelligence process.
```

Comment:
Lower case for 'intelligence', 'threat assessment' and 'risk assessment' and remove 'the' from in front of 'DfE'.